deploy/helm/kuberhealthy/values.yaml

```diff
@@ -149,7 +151,7 @@ check:
     image:
       registry: kuberhealthy
       repository: dns-resolution-check
-      tag: v1.4.2
+      tag: v1.4.3
```

Comment:
be sure this image exists before we merge this to master

Reply:
this was updated for the external-dns-check but not the internal-dns-check :/
https://github.com/Comcast/kuberhealthy/pull/862/files/9e4b4297476f925fcdec45e9b79b390d7e804f1c#diff-a1d3641891145c247db5c0446a75d77fc036d916b0d26d6efe258424f3e7f440R174